test/testapps/BasicTestApp/PropertiesChangedHandlerParent.cshtml

```diff
@@ -1,6 +1,6 @@
 ﻿@addTagHelper "*, BasicTestApp"
 <PropertiesChangedHandlerChild SuppliedValue=@valueToSupply />
-<button onclick=@{ valueToSupply++; }>Increment</button>
+<button onclick=@(x => valueToSupply++)>Increment</button>
```

Comment:
I'd be totally happy to drop support for the `onevent=@{ statement; }` syntax. It's a throwback to the very earliest Blazor prototypes, and no longer appears to be a preferable syntax in any scenario. It's always better to use the lambda syntax because without it, there's no way to receive event args.